src/adapters/postgres/fields-adapter.ts

```
@@ -313,15 +314,22 @@ export class PostgresFieldsService implements IServicelocatorfields {
     }
     for (const data of result) {
       if (
-        ((data?.dependsOn == "" ||
+        ((data?.dependsOn == '' ||
```

ESLint

[error] Replace `''` with `""`

(prettier/prettier)